### ISSUE TITLE
Switched cities not updating properly.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#1]
```
What steps will reproduce the problem?
1. Log in to app while in city A.
2. Change city to city B from playfoursquare.com.
3. Search for any venue in the app

What is the expected output? What do you see instead?
Venues in city B. Instead, it just says "could not complete search". It
doesn't even do a search in city A.

What version of the product are you using? On what operating system?
20090715 / Android

Please provide any additional information below.
Supposedly I can log out of the app to fix this, but I also don't see any
logout functionality anywhere.

```

Original issue reported on code.google.com by `bli...@gmail.com` on 23 Jul 2009 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
Hrmm...

So right now, you can update your city from the preferences screen and upon 
initial 
login your city will update as well. I don't know what I think about 
automatically 
changing the location all the time.

If you update from the site, you currently have to go to preferences and update 
your 
city there as well.

I'm really annoyed by the state mechanisms of city. Basically, when you checkin 
somewhere with city X even if your profile is set to city Y, you should still 
get the 
checkin for city X.

In the end, I've made the ability to switch cities easier but I don't really 
want to 
support auto-city-updates because the pattern itself is already broken.
```

Original comment by `jlapenna` on 3 Aug 2009 at 12:45
- Changed state: **Started**
- Added labels: **Usability**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jlapenna` on 5 Aug 2009 at 1:58
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7a19a2920e.
```

Original comment by `jlapenna` on 29 Aug 2009 at 7:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7a19a2920e.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:47
